# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dotcurve/style.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

More adaptions maybe later.

  - Rene Rebe <rene@exactcode.de>

diff -urN dotcurve-1.0rc1/dotcurve/dotcurve.cpp /home/rene/dotcurve-1.0rc1/dotcurve/dotcurve.cpp
--- dotcurve-1.0rc1/dotcurve/dotcurve.cpp	2005-04-11 19:29:30.000000000 +0200
+++ dotcurve-1.0rc1-mod/dotcurve/dotcurve.cpp	2006-10-05 16:23:18.000000000 +0200
@@ -1560,6 +1560,7 @@
 	switch ( stylehint ) {
 	case SH_TabBar_Alignment: return AlignLeft;
 	case SH_Header_ArrowAlignment: return Right;
+	case SH_LineEdit_PasswordCharacter: return 0x2022; /* unicode bullet char */
 	// to stop gcc complaints
 	default: break;
 	}
